### DISCUSSION TITLE
Fit dynamic animating elements into view

### DISCUSSION THREAD

**Discussion by chivandikwa** [structurizr/ui#42] [Q&A]
It would be cool to zoom in and out automatically as part of animations to always fit things into the view. If two elements next to each other are animated in zooming in would be great and then if a new element that is further comes in the zoom out. This would be great with context diagrams to fit what is in context into the view.

**Comment by simonbrowndotje** [comment]
This will be available in the next release (for dynamic views), via a view/viewset property named `structurizr.zoomOnAnimation`.
